tools/cgeist/driver.cc

```diff
@@ -571,8 +571,11 @@ int main(int argc, char **argv) {
   llvm::DataLayout DL("");
   llvm::Triple gpuTriple;
   llvm::DataLayout gpuDL("");
-  parseMLIR(argv[0], files, cfunction, includeDirs, defines, module, triple, DL,
-            gpuTriple, gpuDL);
+  if (!parseMLIR(argv[0], files, cfunction, includeDirs, defines, module,
+                 triple, DL, gpuTriple, gpuDL)) {
+    llvm::errs() << "parseMLIR failed\n";
```

Comment:
Remove this print and presumably we'll still print the error message from the original clang?

Reply:
Yes, it will be printed. Removed additional `parseMLIR failed` printing